src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -489,10 +549,12 @@ public async Task SynchronizeWithServer_ConnectedModeWithNoIssues_BranchInformat
 
             var serviceProvider = CreateServiceProvider(vsMonitor);
 
+            bindingConfig ??= CreateBindingConfig(SonarLintMode.Connected, "any branch");
+
             var configurationProvider = new Mock<IConfigurationProvider>();
             configurationProvider
                 .Setup(x => x.GetConfiguration())
-                .Returns(new BindingConfiguration(new BoundSonarQubeProject { ProjectKey = SharedProjectKey }, mode, ""));
```

Comment:
Previously the project key was hard-coded inside the test. Now, the caller can pass a `BindingConfiguration` if they need to set the project key or mode specifically.